.github/workflows/rust.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     - name: Configure toolchain
       uses: actions-rs/toolchain@v1
       with:
-        toolchain: nightly-2022-12-02
+        toolchain: nightly-2023-10-01
```

Comment:
We are using nightly-2023-11-10 when building your app for prod. I think you should use the same nightly version